orc8r/cloud/go/obsidian/swagger/handlers/handlers_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handlers_test
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"magma/orc8r/cloud/go/obsidian/swagger"
+	"magma/orc8r/cloud/go/obsidian/swagger/handlers"
+	"magma/orc8r/cloud/go/obsidian/swagger/protos"
+	"magma/orc8r/cloud/go/obsidian/tests"
+	"magma/orc8r/cloud/go/orc8r"
+	swagger_lib "magma/orc8r/cloud/go/swagger"
+	"magma/orc8r/cloud/go/test_utils"
+	"magma/orc8r/lib/go/registry"
+
+	"github.com/labstack/echo"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+)
+
+func Test_GenerateCombinedSpecHandler(t *testing.T) {
```

Comment:
I envision this being a quick sanity-check test, since the handler is a thin wrapper around the existing functionality. I don't think you'll need to do any file I/O